Week-4/Javascript_basics/maps/script.js

```diff
@@ -0,0 +1,21 @@
+const calcWordFrequencies = () => {
+  let question = prompt('Enter the input');
```

Comment:
can refactor question to sentence

Reply:
Fine for better understanding right, will do that

Reply:
yes, variable names should be meaningful, otherwise it can lead to ambiguity.